### PR TITLE
feat: show unknown progress at ui

### PR DESCRIPTION
**Pull request by nd0ut**
JSDK-58

### CODE REVIEW

File: upload-blocks/FileItem/ProgressBar.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { BlockComponent } from '../BlockComponent/BlockComponent.js';
+
+export class ProgressBar extends BlockComponent {
+  _value = 0;
```
Comment (foxeyes) [inline upload-blocks/FileItem/ProgressBar.js:4]: let's use JSdoc here )

File: upload-blocks/FileItem/FileItem.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ import { uploadFile } from '@uploadcare/upload-client';
 import { UiMessage } from '../MessageBox/MessageBox.js';
 import { fileCssBg } from '../svg-backgrounds/svg-backgrounds.js';
 import { customUserAgent } from '../utils/userAgent.js';
+import { ProgressBar } from './ProgressBar.js';
```
Comment (foxeyes) [inline upload-blocks/FileItem/FileItem.js:7]: We've got 2 progress bars in blocks, is that so necessary?

File: upload-blocks/FileItem/FileItem.js [hunk 3]
```diff
@@ -4,6 +4,9 @@ import { uploadFile } from '@uploadcare/upload-client';
...
+
+ProgressBar.reg('file-item-progress-bar');
```
Comment (foxeyes) [inline upload-blocks/FileItem/FileItem.js:9]: Don't you think that we should register blocks in common place? We broke the uniform tag name generation in this case...

**Review by foxeyes — APPROVED** [review]

File: upload-blocks/BlockComponent/BlockComponent.js [hunk 4]
```diff
@@ -238,7 +238,7 @@ export class BlockComponent extends BaseComponent {
           let commonProgress = 0;
           /** @type {String[]} */
           let items = uploadCollection.findItems((entry) => {
-            return !entry.getValue('uploadError');
+            return entry.getValue('uploadError');
```
Comment (nd0ut) [inline upload-blocks/BlockComponent/BlockComponent.js:241]:
```suggestion
            return !entry.getValue('uploadError');
```

**Review by foxeyes — APPROVED** [review]

**Pull request merged**